packaging/rpm/ansible-pylibssh.spec

```diff
@@ -223,8 +223,6 @@ export PYTHONPATH="%{buildroot_site_packages}:${PYTHONPATH}"
 # Fedora:
 %if "%{?fedora:SET}" == "SET"
 %tox -e just-pytest -- \
-  -vv \
-  --installpkg '%{_builddir}/%{pypi_name}-%{upstream_version}/pyproject-wheeldir/%{whl_glob}' \
```

Comment:
@Qalthos why? This would cause a separate wheel build which means that the tests would run against something else..